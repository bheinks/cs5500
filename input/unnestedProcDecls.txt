(*
	Declarations of procedures, no nesting (no errors)
*)

program p;

procedure p1;
begin (* p1 *)
  write(1)
end; (* p1 *)

procedure p2;
begin
  write(2)
end;

begin (* main *)
  write(0)
end.

