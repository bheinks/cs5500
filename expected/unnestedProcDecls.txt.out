

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding p1 to symbol table with type PROCEDURE


>>> Entering new scope...

<<< Exiting scope...

+++ Adding p2 to symbol table with type PROCEDURE


>>> Entering new scope...

<<< Exiting scope...

<<< Exiting scope...

---- Completed parsing ----
